src/Service/PagePurger.php

```diff
@@ -43,6 +51,41 @@ public function purge( Page $page ) {
 	}
 
 	/**
+	 * @since 0.7
+	 *
+	 * @brief Purge multiple pages
+	 *
+	 * Purges all the pages of the Pages object
+	 * by submitting a 'purge' action to the mediawiki
+	 * api with the parameter 'pageids' set to be the
+	 * pages ids in multiple-value seperation.
+	 *
+	 * @param Pages $pages the pages that are going to be purged
+	 *
+	 * @return bool
```

Comment:
Yep, it does ([e.g.](https://en.wikipedia.org/wiki/Special:ApiSandbox#action=purge&format=json&titles=Main_Page%7Cthis_page_really_does_not_exist%7CWikimedia)).